blackmagic/db.py

```diff
@@ -21,25 +21,48 @@
 from images.models import Image as ImageModel
 
 
+class RecoveryImageMissing(Exception):
+    """Image for state recovery is missing. """
```

Comment:
Raised when a client passed an image id for recovery but the image associated with the id wasn't found in the database.